SteamKit2/SteamKit2/Steam/CMClient.cs

```diff
@@ -94,43 +95,8 @@ public IPAddress LocalIP
 
         static CMClient()
         {
-            Servers = new ReadOnlyCollection<IPEndPoint>( new List<IPEndPoint>
-            {
-                new IPEndPoint( IPAddress.Parse( "208.64.200.201" ), 27017 ),
-                new IPEndPoint( IPAddress.Parse( "208.64.200.201" ), 27018 ),
-                new IPEndPoint( IPAddress.Parse( "208.64.200.201" ), 27019 ),
-                new IPEndPoint( IPAddress.Parse( "208.64.200.201" ), 27020 ),
-                new IPEndPoint( IPAddress.Parse( "208.64.200.202" ), 27017 ),
-                new IPEndPoint( IPAddress.Parse( "208.64.200.202" ), 27018 ),
-                new IPEndPoint( IPAddress.Parse( "208.64.200.202" ), 27019 ),
-                new IPEndPoint( IPAddress.Parse( "208.64.200.203" ), 27017 ),
-                new IPEndPoint( IPAddress.Parse( "208.64.200.203" ), 27018 ),
-                new IPEndPoint( IPAddress.Parse( "208.64.200.203" ), 27019 ),
-                new IPEndPoint( IPAddress.Parse( "208.64.200.204" ), 27017 ),
-                new IPEndPoint( IPAddress.Parse( "208.64.200.204" ), 27018 ),
-                new IPEndPoint( IPAddress.Parse( "208.64.200.204" ), 27019 ),
-                new IPEndPoint( IPAddress.Parse( "208.64.200.205" ), 27017 ),
-                new IPEndPoint( IPAddress.Parse( "208.64.200.205" ), 27018 ),
-                new IPEndPoint( IPAddress.Parse( "208.64.200.205" ), 27019 ),
-                new IPEndPoint( IPAddress.Parse( "208.78.164.9" ), 27017 ),
-                new IPEndPoint( IPAddress.Parse( "208.78.164.9" ), 27018 ),
-                new IPEndPoint( IPAddress.Parse( "208.78.164.9" ), 27019 ),
-                new IPEndPoint( IPAddress.Parse( "208.78.164.10" ), 27017 ),
-                new IPEndPoint( IPAddress.Parse( "208.78.164.10" ), 27018 ),
-                new IPEndPoint( IPAddress.Parse( "208.78.164.10" ), 27019 ),
-                new IPEndPoint( IPAddress.Parse( "208.78.164.11" ), 27017 ),
-                new IPEndPoint( IPAddress.Parse( "208.78.164.11" ), 27018 ),
-                new IPEndPoint( IPAddress.Parse( "208.78.164.11" ), 27019 ),
-                new IPEndPoint( IPAddress.Parse( "208.78.164.12" ), 27017 ),
-                new IPEndPoint( IPAddress.Parse( "208.78.164.12" ), 27018 ),
-                new IPEndPoint( IPAddress.Parse( "208.78.164.12" ), 27019 ),
-                new IPEndPoint( IPAddress.Parse( "208.78.164.13" ), 27017 ),
-                new IPEndPoint( IPAddress.Parse( "208.78.164.13" ), 27018 ),
-                new IPEndPoint( IPAddress.Parse( "208.78.164.13" ), 27019 ),
-                new IPEndPoint( IPAddress.Parse( "208.78.164.14" ), 27017 ),
-                new IPEndPoint( IPAddress.Parse( "208.78.164.14" ), 27018 ),
-                new IPEndPoint( IPAddress.Parse( "208.78.164.14" ), 27019 ),
-            } );
+            Servers = new SmartCMServerList();
+            Servers.UseInbuiltList();
```

Comment:
I'd prefer not to have every test start with `Clear()`. We're only doing this once though technically - it's in a static constructor.